src/avatar.tsx

```diff
@@ -69,11 +69,10 @@ const Avatar = ({
         props.backgroundColor ?? (isInverse ? vars.colors.brandHigh : vars.colors.brandLow);
     const textColor = props.textColor ?? (isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand);
     const [imgLoadError, setImgLoadError] = React.useState(false);
-
     React.useEffect(() => {
         setImgLoadError(false); // reset error state when url changes
     }, [src]);
-
+    const borderColor = props.border ? `1px solid ${vars.colors.borderLow}` : 'none';
```

Comment:
please, rename to `border`, you aren't changing the border color only.
```suggestion
    const border = props.border ? `1px solid ${vars.colors.borderLow}` : 'none';
```
alternatively, you can just inline this in the style object below